build/llamafile-convert

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+FILE=$1
+TYPE=${2:-both}
+SCRIPTNAME=$(basename $0)
+
+if [ -z "$FILE" ]; then
+  echo "Usage: $SCRIPTNAME <gguf file or url> [cli|server|both]"
+  exit 1
+fi
+
+# if the file starts with http
+if [[ $FILE == http* ]]; then
```

Comment:
I think glob equals is a bash extension. The POSIX way to do this would be:

```sh
if [ x"$FILE" != x"${FILE#http*}" ]; then  # FILE.startswith("http")
```

Which tests that `$FILE` is not equal to itself stripped of the http prefix.

Even though spaces technically aren't allowed in URLs, it's safer in general to have the quotes here, since it's user input. Many would view that as good from a security standpoint too.